src/azure-cli/azure/cli/command_modules/reservations/_params.py

```diff
@@ -31,3 +31,17 @@ def load_arguments(self, _):
 
     with self.argument_context('reservations catalog show') as c:
         c.argument('reserved_resource_type', arg_type=get_enum_type(ReservedResourceType))
+
+    with self.argument_context('reservations reservation-order calculate') as c:
```

Comment:
`--sku` is also required for this command. Can you list all available values for sku using enum type?

Reply:
I can not list all of the SKUs here. these SKUs are managed by the BigCat team. There are a bunch of SKUs for every ResourceType and they are keeping growing and changing